Source/Core/ExperimentalFeatures.js

```diff
@@ -12,15 +12,12 @@
  *   <li>Experimental flags should be short-lived. Make it clear in the PR what it would take to promote the feature to a regular feature.</li>
  *   <li>To avoid cluttering the code, check the flag in as few places as possible. Ideally this would be a single place.</li>
  * </ul>
- *
- * @private
  */
```

Comment:
I think we need an @experimental tag here.

![image](https://user-images.githubusercontent.com/5172619/140780183-5e8e177a-22a6-409f-8ad8-602485a9c6e0.png)